README.md

```diff
@@ -1,2 +1,18 @@
 # fps-gain-trainer
 Front-end for fps-gain-trainer
+
+## Setup
+### Prerequisites
+- Install `node >= 8.10` and have `npm >= 5.6`.
+- Next you're going to need to install React JS, `cd` into the directory where you want the project to live then run the following commands.
+```
+npx create-react-app app_name
+cd app_name
+``` 
+### Running.
+- Run the `npm start` command in the directory
+- A React homepage should show up if installed correctly.
+
+## Note
+* React JS is unable to run without node installed
```

Comment:
Forgot to talk about the npm install script, would create a lot of frustration for users working on this package for the first time